src/components/auth/AuthButton.js

```diff
@@ -115,20 +136,27 @@ class AuthButton extends React.Component {
   }
 
   render() {
+    const { accessToken } = this.props;
+
     if (this.props.userProfile) {
       const picture = this.props.userProfile.get('picture');
       return (
-        <Popover
-          content={this.logOutMenu()}
-          title={this.popoverTitle(this.props.userProfile)}
-          trigger="click"
-          placement="bottomRight"
-        >
-          <span className="user-button">
-            <Avatar src={picture} icon="user" />
-            <Icon type="caret-down" />
-          </span>
-        </Popover>
+        <React.Fragment>
```

Comment:
Can we not use the `<>foo</>` syntax? Or do you prefer the longer version?

Reply:
TIL

Reply:
I think I still prefer the verbose version but maybe thats just a conditioning thing